Gemfile

```diff
@@ -2,6 +2,11 @@ source "https://rubygems.org"
 
 gemspec name: "ffi-yajl"
 
+group :test do
+  gem "rspec"
```

Comment:
These should go in development so we can keep our standard groups. Also you can remove the github changelog generator